### DISCUSSION TITLE
How do you view commits? I now longer have that option

### DISCUSSION THREAD

**Discussion by johnpankowicz** [community/community#22097] [New to GitHub]
<p>I’m sure there used to be an easy way to view commits, but now I no longer see any menu option to do that. I found out that I can append “/commits/master” to the URL of my repo to get there. But I don’t see any other way.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Just click on the “N commits” under the big green “Code” button. :slightly_smiling_face:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/fd6e9c976a3047ca9e27d00544084a5c9bfe7873.png" data-download-href="/uploads/short-url/A9XMDumyjMxcvrACLMBD4wRzspZ.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/5814919/181068862-c17463f9-5a62-4f66-9002-f19009983a15.png" alt="image" data-base62-sha1="A9XMDumyjMxcvrACLMBD4wRzspZ" width="690" height="221" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/fd6e9c976a3047ca9e27d00544084a5c9bfe7873_2_690x221.png, https://aws1.discourse-cdn.com/github/original/2X/f/fd6e9c976a3047ca9e27d00544084a5c9bfe7873.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/fd6e9c976a3047ca9e27d00544084a5c9bfe7873.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/fd6e9c976a3047ca9e27d00544084a5c9bfe7873_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">813×261 29.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by johnpankowicz** [comment]
<p>Thanks, I wish they had styled that as a link. I’ve been searching for it for 3 days.</p>
